purchases/src/main/kotlin/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -49,10 +55,16 @@ internal class IdentityManager(
             ?: deviceCache.getLegacyCachedAppUserID()
             ?: generateRandomID()
         log(LogIntent.USER, IdentityStrings.IDENTIFYING_APP_USER_ID.format(appUserIDToUse))
-        deviceCache.cacheAppUserID(appUserIDToUse)
-        subscriberAttributesCache.cleanUpSubscriberAttributeCache(appUserIDToUse)
-        deviceCache.cleanupOldAttributionData()
-        invalidateCustomerInfoAndETagCacheIfNeeded(appUserIDToUse)
+
+        val cacheEditor = deviceCache.startEditing()
```

Comment:
This change (chaining the editions) will probably have marginal effects, but I don't think it hurts